### ISSUE TITLE
Gnome Bug: After pressing any system shortcut on keyboard, the overlay window is focused automatically by Gnome.

### ISSUE THREAD

**Issue by titoOdUA** [coderedart/egui_overlay#6]
First of all awesome crate. Thank you.
I have one problem that I can't resolve. I am fairly new to egui and gui development in general, sorry if it is a silly question.

Question: Can I somehow prevent the app from getting focus after using system shortcuts (like changing the keyboard layout)
For example, I am working in the browser, and changing my keyboard (super + space) layout forces focus on my overlay instead of the browser.

Basically, every system shortcut forces focus on the overlay. I guess it is because the overlay is always on top.

But can I somehow prevent it? Maybe to instruct my app to only get focused if some of its elements were clicked?

**Comment by titoOdUA** [comment]
I am on Pop!_OS 22.04 LTS, gnome 42.5, x11

Thank you for the suggestion.
At least it is good to know that it works on other systems/configurations =)
I will try something out.

**Comment by coderedart** [comment]
Then it is probably a gnome bug. Does the same thing happen if you set floating to false?

**Comment by coderedart** [comment]
nice. if you raise a bug report with gone, please post the link here so we can track the progress

**Comment by coderedart** [comment]
It seems some input issues were fixed in the latest gnome 44.3 version. This could be related https://github.com/coderedart/egui_overlay/issues/11

reopen if the issue is still present in gnome 44.3+
